README.md

```diff
@@ -63,6 +68,95 @@ jobs:
           password: '${{ secrets.DbUserPassword }}'
 ```
 
+**Using Azure Service Principal authentication**
+```yml
+jobs:
+  migrate-database:
+    runs-on: [self-hosted, windows-2019]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Flyway
+        uses: actions/setup-flyway@v1
+        with:
+          version: 5.1.4
+
+      - name: Run Flyway Migrations
+        uses: im-open/run-flyway-command@v1.4.0
+        with:
+          db-server-name: 'localhost'
+          db-server-port: '1433'
+          db-name: 'LocalDb'
+          migration-files-path: './src/Database/Migrations'
+          flyway-command: 'migrate'
+          migration-history-table: 'dbo.MigrationHistory'
+          baseline-version: '0'
+          managed-schemas: 'dbo,MyCustomSchema'
+          enable-out-of-order: 'true'
+          use-azure-service-principal: 'true'
+          username: '${{ secrets.AZ_SERVICE_PRINCIPAL_ID }}'
+          password: '${{ secrets.AZ_SERVICE_PRINCIPAL_SECRET }}'
+```
+
+**Using Azure User-Assigned Managed Identity authentication**
+```yml
+jobs:
+  migrate-database:
+    runs-on: [self-hosted, windows-2019]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Flyway
+        uses: actions/setup-flyway@v1
+        with:
+          version: 5.1.4
+
+      - name: Run Flyway Migrations
+        uses: im-open/run-flyway-command@v1.4.0
+        with:
+          db-server-name: 'localhost'
+          db-server-port: '1433'
+          db-name: 'LocalDb'
+          migration-files-path: './src/Database/Migrations'
+          flyway-command: 'migrate'
+          migration-history-table: 'dbo.MigrationHistory'
+          baseline-version: '0'
+          managed-schemas: 'dbo,MyCustomSchema'
+          enable-out-of-order: 'true'
+          use-azure-managed-identity: 'true'
+          azure-msi-client-id: '${{ secrets.AZ_MSI_CLIENT_ID }}'
```

Comment:
`username` is listed as a required property in the Parameters section.

Reply:
Whoops, that's kinda strange. Must be some leftover thing from a long time ago. I'll fix that!